Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -483,7 +484,12 @@ void UIDSys::AddTitle(u64 title_id)
   File::IOFile file(m_file_path, "ab");
 
   if (!file.WriteBytes(&title_id, sizeof(title_id)) || !file.WriteBytes(&uid, sizeof(uid)))
+  {
     ERROR_LOG(IOS_ES, "Failed to write to /sys/uid.sys");
+    return 0;
```

Comment:
UID 0 is used (for the kernel and ES). However, the UID map never has UID 0 in it, since the lowest entry starts with 0x1000.